client/my-sites/domains/domain-management/edit/card/header/primary-domain-button.jsx

```diff
@@ -38,18 +31,35 @@ const PrimaryDomainButton = createReactClass( {
 			label = this.props.translate( 'Make Primary' );
 
 			return (
-				<Button
-					compact
-					className="domain-details-card__make-primary-button"
-					onClick={ this.handleClick }
-				>
+				<Button compact onClick={ this.handleClick }>
 					{ label }
 				</Button>
 			);
 		}
 
 		return null;
-	},
-} );
+	}
+}
```

Comment:
We could simplify this `render()` method a bit:

```javascript
render() {
  const { domain, translate } = this.props;

  if ( domain && ! domain.isPrimary ) {
    return (
      <Button compact onClick={ this.handleClick }>
        { translate( 'Make Primary' ) }
      </Button>
    );
  }

  return null;
}
```

Reply:
done